streamlit_app.py

```
@@ -0,0 +1,51 @@
+
+# Function to capture voice input
+def get_voice_input():
+    r = sr.Recognizer()
+    with sr.Microphone() as source:
+        st.info("Listening... please speak Hinglish")
+        audio = r.listen(source, phrase_time_limit=5)
+    try:
+        text = r.recognize_google(audio)
+        st.success(f"You said: {text}")
+        return text
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)